src/status_im/contexts/communities/actions/airdrop_addresses/view.cljs

```diff
@@ -8,26 +8,23 @@
     [utils.re-frame :as rf]))
 
 (defn- render-item
-  [item]
+  [item _ _ airdrop-address]
   [quo/account-item
    {:account-props item
+    :state         (if (= airdrop-address (:address item)) :selected nil)
```

Comment:
you can use `when'` here since it returns `'nil` if not
`(when (= airdrop-address) (:address-item) :selected)`